Makefile

```diff
@@ -17,7 +17,7 @@ ci:
 	@echo "Run tests"
 	@npm run test
 
-deploy: | decrypt-blackbox configure-docker deploy-image deploy-chart
+deploy: | decrypt-blackbox configure-docker publish-packages deploy-image deploy-chart
```

Comment:
We seldom publish packages to registry in CD flow, since we cannot update a published version, and we don't have automatic version increment setup. Let's do it manually, then there is no need to have secrets.

We should also add publish package instruction in README.

Reply:
gotcha